### PR TITLE
chore: Set up absolute imports

### PR DESCRIPTION
**Pull request by BhanuPratap-Singh-Naruka**
### Task Link

https://www.notion.so/App-Setup-absolute-imports-in-the-app-8f20ac71296f4950a9fe9482c5b14bc1?pvs=4

### Description
Made import shortcut/alias.  
* Before we were importing files like this "../../fileName/nesetedFile"  
* Now we can import like this "fileName/nestedFile"

### Screenshot / Gif (If Any)
Screenshot (21)
Screenshot (20)

### TODO Checklist

- [x] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### CODE REVIEW

**Comment by abhimishra01** [comment]
please shift the absolute imports configurations to app-level so it can cover both `client` & `server` related folders

File: client/tsconfig.json [hunk 1]
```diff
@@ -14,7 +14,14 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "baseUrl": ".",
+   "paths": {
+  "@/*": ["./src/*", "./dist/*", ""],
```
Comment (abhimishra01) [inline client/tsconfig.json:20]: nice

File: client/vite.config.ts [hunk 2]
```diff
@@ -1,7 +1,16 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react-swc";
+import path from "path"
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  resolve: {
+    alias: {
+      "@": path.resolve(__dirname, "./src/"),
```
Comment (abhimishra01) [inline client/vite.config.ts:10]: can you also verify if something like this works ?


```
import { Card } from @components/Card;

```

File: client/tsconfig.json (hunk 1 above)
Comment (BhanuPratap-Singh-Naruka) [reply client/tsconfig.json:20]: Thank you.

File: client/vite.config.ts (hunk 2 above)
Comment (abhimishra01) [reply client/vite.config.ts:10]: well `@/components` feels weird, but for now we will keep it

lets stick to this for now :-
``` import { Card } from components/Card```

**Review by abhimishra01 — APPROVED** [review]

**Pull request merged**